src/library_glfw.js

```diff
@@ -1106,13 +1129,23 @@ var LibraryGLFW = {
     GLFW.hints = GLFW.defaultHints;
     GLFW.windows = new Array()
     GLFW.active = null;
+    // from emscripten_get_device_pixel_ratio
+    GLFW.scale  = window.devicePixelRatio || 1.0;
```

Comment:
This can call that function instead of copying the code:
```suggestion
    GLFW.scale  = _emscripten_get_device_pixel_ratio();
```
Also you'll need to add a dep, on line 1124,
```js
glfwInit__deps: ['emscripten_get_device_pixel_ratio'],
```